xarray/core/dataset.py

```diff
@@ -307,6 +308,7 @@ class Dataset(Mapping, ImplementsDatasetReduce, BaseDataObject,
     coordinates used for label based indexing.
     """
     groupby_cls = groupby.DatasetGroupBy
+    rolling_cls = rolling.DatasetRolling
```

Comment:
I think this should be private (I know `groupby_cls` is public, but I think that is a mistake)